sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -0,0 +1,290 @@
+<template>
+  <!-- This component is shared between ministry and student users -->
+  <p class="category-header-large color-blue">Applications</p>
+  <DataTable
+    :value="applicationAndCount.applications"
+    :lazy="true"
```

Comment:
lazy and paginator can be declared static.

Reply:
added comment above